src/StreamJsonRpc.Tests/JsonRpcTests.cs

```diff
@@ -303,6 +303,14 @@ public void Attach_Null_Throws()
     }
 
     [Fact]
+    public async Task CannotCallMethodsOnSystemObject()
+    {
+        await Assert.ThrowsAsync<RemoteMethodNotFoundException>(() => this.clientRpc.InvokeAsync(nameof(object.ToString)));
+        await Assert.ThrowsAsync<RemoteMethodNotFoundException>(() => this.clientRpc.InvokeAsync(nameof(object.GetHashCode)));
+        await Assert.ThrowsAsync<RemoteMethodNotFoundException>(() => this.clientRpc.InvokeAsync(nameof(object.GetType)));
```

Comment:
I felt this was an adequate enough sampling of methods to demonstrate that they weren't available. The product would have to do something rather specific to allow some System.Object methods and not others.